### PR TITLE
libmustache 0.4.4

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/libmustache.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Libmustache < Formula
-  desc "C++ implementation of Mustache."
+  desc "C++ implementation of Mustache"
   homepage "https://github.com/jbboehr/libmustache"
   url "https://github.com/jbboehr/libmustache.git",
-      :tag => "v0.4.3",
-      :revision => "081952948f6e50f0eeb116fd652585813f31a2ab"
+      :tag => "v0.4.4",
```
Comment (SMillerDev) [inline Formula/libmustache.rb]: might be better to switch to release tars while we're at it?

Comment (ilovezfs) [reply Formula/libmustache.rb]: It looks like it has git submodules. Not sure if they're needed.

**Comment by ilovezfs** [comment]
```
checking for /private/tmp/libmustache-20180305-92823-sl1n3e/libmustache-0.4.4/spec/specs/comments.yml... no
configure: error: in `/private/tmp/libmustache-20180305-92823-sl1n3e/libmustache-0.4.4':
configure: error: mustache spec missing (--without-mustache-spec to disable)
See `config.log' for more details
```
Shall I pass `--without-mustache-spec` or go back to using tag?

**Comment by SMillerDev** [comment]
Looks like a pretty essential part of the lib. Back to tag it is then 😢

**Pull request closed without merging**